<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>867</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(46, 52, 54);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cantarell</family>
      <pointsize>20</pointsize>
      <weight>9</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 oblique 20pt &quot;Cantarell&quot;;
color: white</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Eventos&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>771</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QPushButton" name="button_uni_create">
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;Cantarell&quot;;
color: white</string>
         </property>
         <property name="text">
          <string>Cadastro de festa universitária</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_uni_update">
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;Cantarell&quot;;
color: white</string>
         </property>
         <property name="text">
          <string>Atualização de festa universitária</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_uni_services">
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;Cantarell&quot;;
color: white</string>
         </property>
         <property name="text">
          <string>Serviços das festas universitárias</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QPushButton" name="button_wed_create">
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;Cantarell&quot;;
color: white</string>
         </property>
         <property name="text">
          <string>Cadastro de festa de casamento</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_wed_update">
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;Cantarell&quot;;
color: white</string>
         </property>
         <property name="text">
          <string>Atualização de festa de casamento</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_wed_services">
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;Cantarell&quot;;
color: white</string>
         </property>
         <property name="text">
          <string>Serviços das festas de casamento</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="button_reports">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>460</y>
      <width>278</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Cantarell&quot;;
color: white</string>
    </property>
    <property name="text">
     <string>Relatórios</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_search_events">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>420</y>
      <width>278</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Cantarell&quot;;
color: white</string>
    </property>
    <property name="text">
     <string>Pesquisar festas</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>2</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 50 oblique 20 &quot;Cantarell&quot;;
color: white</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Festas Universitárias&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>90</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>2</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 50 oblique 20 &quot;Cantarell&quot;;
color: white</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Festas de Casamento&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>867</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
